meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 # which must contain only alphanumeric symbols,'-' and '_'.
 # Any other symbols would be skipped in parser.
 #
-# Following targets would be used to perform default build task:
-OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
+# Following targets would be used to perform default build task.
+OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
```

Comment:
On Wed, 2016-03-30 at 01:43 -0700, Alexander D. Kanevskiy wrote:
> In meta-ostro/conf/distro/include/ostroproject-ci.inc:
> 
> > @@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
> >  # which must contain only alphanumeric symbols,'-' and '_'.
> >  # Any other symbols would be skipped in parser.
> >  #
> > -# Following targets would be used to perform default build task:
> > -OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
> > +# Following targets would be used to perform default build task.
> > +OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
> 
> git grep '${DISTRO}-image'

Someone who doesn't know about ostroproject-ci.inc will also not know
that he has to grep with that term.